util/run_manual_tests.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+import subprocess
+from pathlib import Path
+
+import hjson
+import typer
+from pluralizer import Pluralizer
+from rich.console import Console
+from rich.markdown import Markdown
+from rich.prompt import Confirm
+from rich.text import Text
+
+pluralize = Pluralizer().pluralize
+
+
+def get_tty():
+    """Returns the terminal that we should use.
+
+    When this script is run directly, the terminal should be the terminal of the process.
+    When this script is run by bazel, the terminal should be the terminal of the bazel
+    client. Note that we cannot simply traverse the process tree, since this script is
+    run by the bazel server, which could be started from a different terminal.
+    """
+    term = subprocess.check_output(
+        ["/bin/ps", "-p", f"{os.getpid()}", "-o", "tty="]).decode().strip()
+    if term == "?":
+        res = subprocess.check_output(["/bin/ps", "-axo",
+                                       "tty=,cmd="]).decode()
+        m = re.search(r"^(?P<term>[\w/]*) *bazel.*test.*$", res, re.MULTILINE)
+        if not m:
+            raise RuntimeError("Could not find the terminal")
+        term = m["term"]
+    return f"/dev/{term}"
+
+
+# Set the terminal manually since this test is interactive.
+# This is really a hack since bazel does not support interactive commands.
+stdin = open(get_tty(), "r")
+stdout = open(get_tty(), "w")
```

Comment:
nit: I assume this script captures all the in/out to the Bazel log. It would be good to unconditionally print the path to the log if possible, or are more details in the documentation to how we are planning to use the log to capture the sign-off. 

This makes me think we should document also how to tie the log to a commit hash or git tag.

Reply:
I haven't checked what the log file contains but an easy workaround would be to use `--test_output=streamed` and redirect to a file. If we feel strongly about the log, we can probably enhance this script to write to both `fd`s.

We should think about the second point a little more but assuming we are not worried about authenticity, can putting this in the name an option, e.g. `rom_signoff_tests_<committish>.log`?